cmd/fyne_demo/main.go

```diff
@@ -251,14 +251,14 @@ func makeNav(setTutorial func(tutorial tutorials.Tutorial), loadPrevious bool) f
 	return container.NewBorder(nil, themes, nil, nil, tree)
 }
 
-func shortcutFocused(s fyne.Shortcut, w fyne.Window) {
+func shortcutFocused(s fyne.Shortcut, a fyne.App, w fyne.Window) {
```

Comment:
I wonder if it perhaps makes more sense to pass in the focused object now that we don't need the window for other purposes?

Reply:
Sure, just did a mechanical fix, patch sent.